### PR TITLE
First draft of functions lab

### PR DESCRIPTION
**Pull request by mdippery**
cc: @AnnJohn @JimCampagno

### CODE REVIEW

File: MyPlayground.playground/Pages/main.xcplaygroundpage/Contents.swift [hunk 1]
```diff
@@ -3,35 +3,45 @@
...
  */
 // write your code here
 
 
 
 
 /*: question2
- ### 2. Another question about functions.. weeee
+ ### 2. Write a function that declares a constant inside the function's body and prints it to the console. Then call it to see your string printed to the playground console.
  */
```
Comment (JimCampagno) [inline MyPlayground.playground/Pages/main.xcplaygroundpage/Contents.swift:21]: Being slightly more explicit:

> Write your own function, where inside your function's body you declare a constant and then print that constant to the console. Call this function to see your string printed to the playground console.

File: MyPlayground.playground/Pages/main.xcplaygroundpage/Contents.swift [hunk 2]
```diff
@@ -3,35 +3,45 @@
...
+ ### 4. Now call the function you wrote in Question 3 using a variable or constant instead of a string literal. What do you expect to see in the console? Try passing in a _variable_ you declared (using `var`) as an argument. Then change that variable's value and call your function again. What do you see in the console?
+ */
+// write your code here
+
+
+
+
+/*: question5
+ ### 5. Write a function that declares a _variable_ inside the function's body. Print that variable to the console, then assign it to a new variable and print it again. Call your function several times. What do you think happens each time you call your function? Will it print the same thing?
  */
```
Comment (JimCampagno) [inline MyPlayground.playground/Pages/main.xcplaygroundpage/Contents.swift:45]: Similar to the my above comment:

> Write your own function, where inside the function's body you declare a variable (of any type you like). Print out this variable to the console from within the function. After you print the variable, assign it a new value on the next line. Print out the variable again (after where you assigned it a new value). What do you think prints to the console when you call this function?

**Comment by mdippery** [comment]
@JimCampagno, I cleaned up those questions.

**Comment by JimCampagno** [comment]
@mdippery Thanks.

[nerdy guitar video](https://www.youtube.com/watch?v=SIak9tw_XMw)

**Pull request merged**